src/mono/mono/component/debugger-agent.c

```diff
@@ -7051,6 +7051,10 @@ vm_commands (int command, int id, guint8 *p, guint8 *end, Buffer *buf)
 		buffer_add_moduleid (buf, mono_get_root_domain (), assembly->image);
 		break;
 	}
+	case MDBGPROT_CMD_GET_ENC_CAPABILITIES: {
+		buffer_add_string (buf, "Baseline AddMethodToExistingType AddStaticFieldToExistingType NewTypeDefinition ChangeCustomAttributes");
```

Comment:
Let's share this string between the debugger and the managed get capabilities API.

Reply:
Done :)